src/components/code-editor/code-editor.tsx

```diff
@@ -20,8 +21,6 @@ import 'codemirror/addon/fold/brace-fold';
 import jslint from 'jsonlint-mod';
 
 /**
- * Currently this component support syntax highlighting for `javascript`,
- * `json` and `typescript` formats.
```

Comment:
Seems a little pointless to list which languages we support syntax highlighting for, when that list is identical to the list of languages we support at all… 😄

Reply:
Agree! Missed this comment.